appmap/_implementation/env.py

```diff
@@ -97,7 +101,7 @@ def _configure_logging(self):
                 'appmap': {
                     'level': log_level,
                     'handlers': ['default'],
-                    'propagate': False
+                    'propagate': True
```

Comment:
What motivated this change? IIRC, I set it to `False` because allowing the messages to propagate would cause them to be sent to the application's loggers, too. This may even have caused test failures.

Reply:
If it doesn't propagate then it doesn't get to the pytest handler, so we can't check for the message.

I don't think it's necessarily a bad thing that appmap output gets sent to the application's loggers; unless it does break a test, of course. Perhaps we should add a flag to disable propagation?

Reply:
Ok, that makes sense. We can go with this for now, and add a flag in the future, if necessary.